server.js

```diff
@@ -26,6 +26,7 @@ app.use(bodyParser.urlencoded({extended: true}));
 
 app.get('/api/comments', function(req, res) {
   fs.readFile(COMMENTS_FILE, function(err, data) {
+    if (err) console.log(err);
```

Comment:
@zpao Thank you for the input. I completely agree that it is not a huge deal as it's not meant to be production ready code, and that adding extensive error handling may result in unneeded code bloating. I made some edits based on your suggestions.